.circleci/config.yml

```diff
@@ -20,7 +20,7 @@ jobs:
         # day to keep the cache size down.
       - run:
           name: Build the container ( Next 3 steps )
-          command: I="image-$(date +%j).gz"; if [[ -e ~/docker/$I ]]; then echo "Loading $I"; pigz -d -c ~/docker/$I | docker load; fi
+          command: I="image-$(date +%j).gz"; if [[ -e ~/addons-frontend/docker/$I ]]; then echo "Loading $I"; pigz -d -c ~/addons-frontend/docker/$I | docker load; fi
```

Comment:
Since `$I` is referenced across multiple commands now, can you call it `$IMG`? That will make it easier to search for when trying to figure it out.